lib/core/reporters/no-passes.js

```diff
@@ -9,8 +9,25 @@ axe.addReporter('no-passes', function(results, options, callback) {
 	options.resultTypes = ['violations'];
 
 	var out = helpers.processAggregate(results, options);
+	var orientation =
```

Comment:
Is it worth moving this into a `getScreenOrientation` function since it's duplicated across files?

Again, not a big deal IMO, just a thought.